### ISSUE TITLE
Remote harvester does not work

### ISSUE THREAD

**Issue by moon2me** [Spare-Network/spare-blockchain#50]
you can't set up a remote harvester.

also if you try to use the command line it always tries to start the ui.

**Comment by Sector-Network** [comment]
Make sure you are under resources/app.asar.unpacked/daemon

**Comment by moon2me** [comment]
I was in an admin powershell, I am using windows, the first time I couldn't get it to sync then with the new version where the users directory is .spare-blockchain worked. Then I went on to try to set up harvesting on the other machines.

.\spare configure --set-farmer-peer 192.x.x.x:9447

however it will try to load the gui when I run this command or any of the others, I just used the config file and added all the settings manually copied the ca directory, if I try to start the harvester it will not start the daemon.

**Comment by moon2me** [comment]
Ok have it working on all but 1 machine, issue with the port when looking at log.

**Comment by markjbh** [comment]
can i ask what you did to get it working? i have tried doing same\similar to chia with the yaml setup but cant get the famer to see harvesters. thanks

**Comment by moon2me** [comment]
Yes I got it working, I didn't trust so I set the log level to info on the harvester, you can then see the entries adding the plots and the communication.

.\spare init -c C:\sparessl\ca
.\spare plots add -d p:\plots - the config file was weirdly spaced before I added an entry. Now I modified the config file with any other plot directories.
.\spare configure --set-farmer-peer 192.xxx.xxx.xxx:9447

All but 1 machine worked I need to see what else is using port 59400 on that machine.
